modules/overrides/userconfigurableapi/client.go

```diff
@@ -42,74 +47,89 @@ var (
 type UserConfigurableOverridesClientConfig struct {
 	Backend string `yaml:"backend"`
 
+	// ConfirmVersioning is enabled when creating the backend client. If versioning is disabled no
+	// checks against concurrent writes will be performed.
+	ConfirmVersioning bool `yaml:"confirm_versioning"`
```

Comment:
I considered placing this on `gcs.Config` instead but I'm worried this would conflict with our existing bucket configuration. For storing traces you don't need versioning, so setting this to default true would cause a lot of issues.

Reply:
Agree with the reasoning here 👍